overlays/bases/rbac-roles/kustomization.yaml

```diff
@@ -7,6 +7,5 @@ resources:
   - base/prometheus/prometheus.ClusterRole.yaml
   - base/cadvisor/cadvisor.ClusterRoleBinding.yaml
   - base/cadvisor/cadvisor.ClusterRole.yaml
-  - base/cadvisor/cadvisor.DaemonSet.yaml
```

Comment:
IIRC i put this is here because I'm not sure cadvisor really works without the cluster role thingo (could be wrong though? this was a bit of a shot in the dark) do you mind explaining this move a bit?

Reply:
This isn't removing the cluster role, but if someone wants to make a change to something inside cadvisor (the image for example) they shouldn't need to include `..bases/rbac` in their kustomization, only deployments.

Reply:
oh I think I might have misunderstood this back then a bit then

Reply:
TIL this after trying to find out what my kustomizations weren't generating as expected :-)